packages/widgetbook_core/lib/src/navigation_tree/bloc/navigation_bloc.dart

```diff
@@ -69,7 +67,10 @@ class NavigationBloc extends Bloc<NavigationEvent, NavigationState> {
     SelectNavigationNodeByPath event,
     Emitter<NavigationState> emit,
   ) {
-    final targetNode = _filterNodesByPath(event.path);
+    final uri = Uri.parse(event.path);
+    final finalPath = uri.queryParameters['path'];
+    if (finalPath == null) return;
+    final targetNode = _filterNodesByPath(finalPath);
```

Comment:
this was kind of blocking me, so I added this quick fix, but also created an issue #446 to track it. 

Should we handle this in a different way?